cloudflare_utils.py

```diff
@@ -1,43 +1,42 @@
 #!/usr/bin/env python3
 
-import ipaddress
 import dns
+import ipaddress
 import sys
 
-#  TODO: fetch it from https://www.cloudflare.com/ips-v4 instead
-cloudflare_ip_ranges = [
-    "103.21.244.0/22",
-    "103.22.200.0/22",
-    "103.31.4.0/22",
-    "104.16.0.0/12",
-    "108.162.192.0/18",
-    "131.0.72.0/22",
-    "141.101.64.0/18",
-    "162.158.0.0/15",
-    "172.64.0.0/13",
-    "173.245.48.0/20",
-    "188.114.96.0/20",
-    "190.93.240.0/20",
-    "197.234.240.0/22",
-    "198.41.128.0/17"
-]
+import requests
+
+
+def get_cloudflare_ip_ranges():
```

Comment:
Would be great to have a bit more logging, e.g. `[*] Retrieving Cloudflare IP ranges from https://www.cloudflare.com/ips-v4`

Reply:
That is true. Informing the user of the current action is better.